### ISSUE TITLE
Can I completely shutdown telnet service, if device is vulnerable?

### ISSUE THREAD

**Issue by Areso** [JacobMisirian/DblTekGoIPPwn#1]
Sadly, there are no hints about telnet service settings in user's manual (except options for Port Transparency/DMZ, which is not about the subject).

**Comment by JacobMisirian** [comment]
I do not believe you can fully turn off the service, although I am not an expert in operating these devices. However, closing down the port using iptables (or NAT on your router) is definitely an option. Better yet, update to the latest firmware. Even though DblTek has not put out a firmware that completely removes this backdoor nonsense, the latest one's challenge response is complex enough that someone like me is too lazy to reverse engineer it.
